### PR TITLE
Chore: Rename payment builder action

### LINKED ISSUE #2493: Payment builder feature global change
## Description

To streamline our features for users, the 'Payment builder' feature is being renamed 'Advanced payment'.

All references globally including action types, metadata, activity feeds and extensions need to be renamed. 

Some examples of where the payment builder is currently referenced:

Actions dropdowns:
image
 
 Action:
image

### PR DESCRIPTION
**Pull request by mmioana**
Rename Payment Builder action to Advanced Payment

## Description

Refactor 'Payment Builder' action to 'Advanced payment'

## Testing

* Step 1. Create new payment builder action 
* Step 2. Check naming is 'Advanced payment'

Resolves #2493

### CODE REVIEW

**Comment by rdig** [comment]
Is this ready for review? _(Because It's still in draft mode)_

**Comment by mmioana** [comment]
I put it in draft mode as I didn't manage to test if the correct string shows up after action creation in the actions list. 
Today I'm going to check with Kerry with a different decision method

**Review by rdig — APPROVED** [review]
Nicely done

**Review by jakubcolony — APPROVED** [review]
Nice one @mmioana 👍

**Comment by rdig** [comment]
@mmioana please wait for three approvals before merging _(we're not yet enforcing this rule, just testing it out)_

**Pull request merged**
